applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -163,26 +184,28 @@ static void public_broadcast_features_set(uint8_t *features)
 }
 #endif /* (CONFIG_AURACAST) */
 
-static int adv_create(void)
+int broadcast_source_ext_adv_populate(struct bt_data *adv_buf, size_t adv_buf_vacant)
 {
 	int ret;
 	uint32_t broadcast_id = 0;
+	size_t adv_buf_cnt = 0;
+	size_t brdcst_name_size;
 
-	/* Broadcast Audio Streaming Endpoint advertising data */
-	NET_BUF_SIMPLE_DEFINE_STATIC(brdcst_id_buf, BT_UUID_SIZE_16 + BT_AUDIO_BROADCAST_ID_SIZE);
-	/* Buffer for Appearance */
-	NET_BUF_SIMPLE_DEFINE_STATIC(brdcst_appearance_buf, BT_UUID_SIZE_16);
-	/* Buffer for manufacturer ID */
-	NET_BUF_SIMPLE_DEFINE_STATIC(manufacturer_id_buf, BT_UUID_SIZE_16);
-	/* Buffer for Public Broadcast Announcement */
-	NET_BUF_SIMPLE_DEFINE_STATIC(base_buf, 128);
-
-	net_buf_simple_add_le16(&manufacturer_id_buf, CONFIG_BT_DEVICE_MANUFACTURER_ID);
+	if (IS_ENABLED(CONFIG_BT_AUDIO_USE_BROADCAST_NAME_ALT)) {
+		brdcst_name_size = sizeof(CONFIG_BT_AUDIO_BROADCAST_NAME_ALT) - 1;
+		memcpy(brdcst_name_buf, CONFIG_BT_AUDIO_BROADCAST_NAME_ALT, brdcst_name_size);
+	} else {
+		brdcst_name_size = sizeof(CONFIG_BT_AUDIO_BROADCAST_NAME) - 1;
+		memcpy(brdcst_name_buf, CONFIG_BT_AUDIO_BROADCAST_NAME, brdcst_name_size);
+	}
 
-	ext_ad[0].data_len = manufacturer_id_buf.len;
-	ext_ad[0].type = BT_DATA_UUID16_SOME;
-	ext_ad[0].data = manufacturer_id_buf.data;
+	ret = bt_mgmt_adv_buffer_put(adv_buf, adv_buf_vacant, &adv_buf_cnt, BT_DATA_BROADCAST_NAME,
+				     brdcst_name_size, &brdcst_name_buf[0]);
```

Comment:
```suggestion
				     brdcst_name_size, brdcst_name_buf);
```
Should be the same, I believe

Reply:
Same for the others